process/block/shardblock.go

```diff
@@ -154,6 +154,10 @@ func (sp *shardProcessor) ProcessBlock(
 		return err
 	}
 
+	log.Info(fmt.Sprintf("starting process block with round %d and nonce %d\n",
```

Comment:
I think these log.Info's are not really needed. Put into debug and for internal testing we could start with log flags of debug

Reply:
Done